_source/_posts/2023-07-28-scim-workshop.md

```diff
@@ -1331,3 +1331,33 @@ If you have followed along with the workshops so far, you now have a Todo applic
 | 4. [Enterprise-Ready Workshop: Terraform](/blog/2023/07/28/terraform-workshop) |
 
 Follow us on [Twitter](https://twitter.com/oktadev) and subscribe to our [YouTube](https://www.youtube.com/c/oktadev) channel to get notified about exciting new content. Please comment below if you have any questions or want to share problems you have used SCIM to solve or are planning to solve. As all our sample projects and SDKs are open source, we invite you to initiate a pull request if you see something you want us to improve. Thank you! 
+
+## Supplemental
```

Comment:
Can we find a way to sneak in keywords here? such as "Enhance your SCIM implementation" or "Manage multi-tenancy with SCIM"?